client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,162 @@
+import { AppBar, Toolbar, Typography, Badge, IconButton, Drawer, Box, Grid } from '@material-ui/core';
+
+import { Link } from 'react-router-dom';
+import MenuIcon from '@material-ui/icons/Menu';
+import React, { useState, useEffect } from 'react';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+import useStyles from './useStyles';
+import PetsIcon from '@material-ui/icons/Pets';
+import { User } from '../../interface/User';
+
+interface Props {
+  logOut: string;
+  profile: string;
+  mySitter: string;
+  becomeSitter: string;
+  messages: string;
+  loggedIn: boolean;
+  loggedInUser: User;
+  open: boolean;
+}
+
+const Navbar = ({
+  logOut,
+  profile,
+  mySitter,
+  becomeSitter,
+  messages,
+  loggedInUser,
+  loggedIn,
+  open,
+}: Props): JSX.Element => {
+  const classes = useStyles();
+
+  const [state, setState] = useState({
```

Comment:
I second Ugur's comment below. Also, maybe consider giving the state variable a more descriptive name?

Reply:
given descriptive name to both states